### ISSUE TITLE
Lodash Vite error - `Two output files share the same path but have different contents` 

### ISSUE THREAD

**Issue by Jma353** [tajo/ladle#102]
My team is interested in using this project for React component development and to build a component library.

## The Problem
When onboarding to this project, we hit the following errors when running `npm run ladle serve`:

```
9:18:08 PM [vite] error while updating dependencies:
Error: Build failed with 2 errors:
error: Two output files share the same path but have different contents: node_modules/.vite/processing/lodash_cloneDeep.js.map`
error: Two output files share the same path but have different contents: node_modules/.vite/processing/lodash_cloneDeep.js
```

The reason for this is this project depending on and importing `lodash.clonedeep`. Our project uses and imports `lodash/cloneDeep`. 

**Within the `ladle` project, `optimizeDeps.include` is configured to optimize `lodash.clonedeep`.** This leads to an error when building via Vite. This problem has been documented before in [this StackOverflow post.](https://stackoverflow.com/questions/71354955/sveltekit-or-esbuild-error-two-output-files-share-the-same-path-but-have-differ)

If I comment out this line in the built `ladle` project locally, our Ladle component previews work seamlessly.

## Ideas
`lodash/**` import usage is common in projects today, and can easily conflict with the optimized dependencies configured currently in the `ladle` project. I see 2 alternatives: 

1. Remove `lodash.**` dependency optimizations from `ladle`. 
2. Enable users of `ladle` to overwrite `optimizeDeps.include`, vs. strictly enabling _adding_ to this list (this is a restriction in the configuration API today to my understanding). 

## Finally....
_I'm happy to help push a solution to this problem forward. I wanted to get a sense for what solution we prefer before I cut a PR. Would love thoughts!_

cc: @tajo

**Comment by tajo** [comment]
We might try to remove it from `optimizeDeps.include` but is it solving the issue?

I think the best solution might be to inline the source code of lodash.clonedeep into ladle.

**Comment by Jma353** [comment]
If you wanna inline, that would be great. I'm on-board with that.

I commented out the `optimizeDeps.include` line for `lodash.clonedeep` in my `node_modules` and it works perfectly locally. 

Can I cut a PR for this? Or do you wanna do it @tajo.

**Comment by tajo** [comment]
> Can I cut a PR for this? Or do you wanna do it @tajo.

Yea. I prefer inlining.

**Comment by Jma353** [comment]
@tajo - fix here #105

**Comment by Jma353** [comment]
Closing this b/c I fixed it in my PR.
